addons/mail/models/mail_activity.py

```diff
@@ -389,6 +393,28 @@ def _compute_display_name(self):
             name = record.summary or record.activity_type_id.display_name
             record.display_name = name
 
+    # ------------------------------------------------------
+    # Notifications
+    # ------------------------------------------------------
+
+    def _notify_owner_activity(self, unlink=True):
```

Comment:
I'm not a fan of the `owner` terminology. Here is a suggestion: 

```suggestion
    def _notify_activity_thread(self, unlink=True):
```